docs/release_notes/pending_release.rst

```diff
@@ -85,3 +79,7 @@ Utilities
 
 * Fix ``xaitk_saliency.utils.masking.weight_regions_by_scalar`` to not upcast
   the data type output when ``inv_masks is True``.
+
+* Fix ``xaitk_saliency.utils.gen_coco_sal.gen_coco_sal`` to filter out images
+  in the COCO dataset that do not have any detections associated with them.
+  This fixes the info log to print the correct number of images.
```

Comment:
I don't think we need to add this "fix" note since we just introduced the tool within this same "release." I.e. it's a little odd to introduce that we added `gen_coco_sal` (the new L39 above) and then fixed the very same thing.